package.json

```diff
@@ -74,5 +74,6 @@
   "resolutions": {
     "@storybook/blocks@^7.6.17": "patch:@storybook/blocks@npm%3A7.6.17#./.yarn/patches/@storybook-blocks-npm-7.6.17-481e2fff02.patch",
     "@storybook/blocks@7.6.17": "patch:@storybook/blocks@npm%3A7.6.17#./.yarn/patches/@storybook-blocks-npm-7.6.17-481e2fff02.patch"
-  }
+  },
+  "packageManager": "yarn@3.5.0"
```

Comment:
why to add this?

Reply:
It was a requirement from turborepo, not sure why though.